languagetool-core/src/main/java/org/languagetool/CsvToXml.java

```diff
@@ -0,0 +1,132 @@
+import javax.xml.transform.dom.DOMSource;
+import javax.xml.transform.stream.StreamResult;
+
+import org.w3c.dom.Document;
+import org.w3c.dom.Element;
+
+public class CsvToXml {
+
+  public static void main(String[] args) {
+    Scanner scanner = new Scanner(System.in);
+    System.out.print("Choose interface (To choose command line type 'cl', to choose UI type 'ui'): ");
+    System.out.flush();
+    String screen = scanner.nextLine();
+
+    if ("cl".equals(screen)) {
+      System.out.print("Enter a file name: ");
+      System.out.flush();
+      String filename = scanner.nextLine();
+      System.out.print("Enter character to seperate csv file: ");
+      System.out.flush();
+      String splitBy = scanner.next();
+      csvToBitext(filename, splitBy, "");
+    } else if ("ui".equals(screen)) {
+      JFrame.setDefaultLookAndFeelDecorated(true);
+      JDialog.setDefaultLookAndFeelDecorated(true);
+      JFrame frame = new JFrame("User Interface");
+      frame.setPreferredSize(new Dimension(400, 100));
+      frame.setLayout(new FlowLayout());
+      frame.setDefaultCloseOperation(JFrame.EXIT_ON_CLOSE);
+      JLabel text = new JLabel("Split words by:");
+      JTextField split = new JTextField(5);
+      JButton btnFile = new JButton("Select Csv File");
+      JButton btnOk = new JButton("Ok");
+      JFileChooser jfc = new JFileChooser(FileSystemView.getFileSystemView().getHomeDirectory());
+      btnFile.addActionListener((ActionEvent ae) -> {
+        jfc.setDialogTitle("Select csv file");
+        jfc.setAcceptAllFileFilterUsed(false);
+//                FileNameExtensionFilter filter = new FileNameExtensionFilter("csv");
+//                jfc.addChoosableFileFilter(filter);
+
+        int returnValue = jfc.showOpenDialog(null);
+//                if (returnValue == JFileChooser.APPROVE_OPTION) {
+//                    
+//                }
+      });
+      btnOk.addActionListener((ActionEvent ae) -> {
+        csvToBitext(jfc.getSelectedFile().getPath(), split.getText(), "");
+      });
+      frame.add(btnFile);
+      frame.add(text);
+      frame.add(split);
+      frame.add(btnOk);
+      frame.pack();
+      frame.setVisible(true);
+    }
+  }
+
+  public static void csvToBitext(String csvFile, String cvsSplitBy, String line) {
+
+    try (BufferedReader br = new BufferedReader(new FileReader(csvFile))) {
+      DocumentBuilderFactory docFactory = DocumentBuilderFactory.newInstance();
+      DocumentBuilder docBuilder = docFactory.newDocumentBuilder();
+
+      Document doc = docBuilder.newDocument();
+      Element rootElement = doc.createElement("example");
+      doc.appendChild(rootElement);
+
+      Element example00 = doc.createElement("example00");
+      rootElement.appendChild(example00);
+
+      Element example01 = doc.createElement("example01");
+      rootElement.appendChild(example01);
+
+      example00.setAttribute("lang", "en");
+
+      example01.setAttribute("lang", "ru");
```

Comment:
Not sure, what exactly is this code supposed to do, why is `ru` hard coded here? Should this be a test case instead?